frontend/src/tests/instructor-permission-test.js

```diff
@@ -16,6 +16,86 @@ export function instructorsPermissionTests(api) {
     const { apiGET, apiPOST } = api;
     let session = null;
 
+    const executeWithInstOnlyUser = async (action) => {
+        const newInstOnlyUser = {
+            utorid: "test",
+            roles: ["instructor"],
+        };
+        let respAddInstOnlyUser = await apiPOST(
+            `/debug/users`,
+            newInstOnlyUser
+        );
+        expect(respAddInstOnlyUser).toHaveStatus("success");
+
+        const respOriginalUser = await apiGET(`/debug/active_user`);
+        expect(respOriginalUser).toHaveStatus("success");
+        const originalUser = {
+            utorid: respOriginalUser.payload.utorid,
+            roles: respOriginalUser.payload.roles,
+        };
+
+        let respSwitchToInstOnlyUser = await apiPOST(
+            `/debug/active_user`,
+            newInstOnlyUser
+        );
+        expect(respSwitchToInstOnlyUser).toHaveStatus("success");
+
+        await action();
+
+        let respSwitchBackUser = await apiPOST(
+            `/debug/active_user`,
+            originalUser
+        );
+        expect(respSwitchBackUser).toHaveStatus("success");
+    };
+
+    const cantUpdateTestFrame = async (
```

Comment:
Prefer `function cantUpdateTestFrame` to `const cantUpdateTestFrame = () => {}`.